kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/ContentCardList.vue

```diff
@@ -37,7 +37,14 @@
           disabled, tabindex, is-leaf class set here to hack making the card not clickable
           if you're trying to make the card clickable remove these properties
         -->
+        <AccessibleFolderCard
```

Comment:
@AlexVelezLl See https://github.com/learningequality/kolibri/pull/12418/files#r1691947415. This will be handled by card grid component in the future.

Reply:
@AlexVelezLl, `ContentCardList.vue` changes will be reverted before we merge the feature `introduce-cards` to `develop`. The aim of the current task is only to implement the card itself. I asked @AllanOXDi to find a place that would allow us to preview, but it's really not meant for production.

Reply:
^ This applies to all upcoming work to `introduce-cards` feature branch

Reply:
Got it hehe, thank you @MisRob!